source/includes/aggregation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$db = $client->sample_restaurants;
+$collection = $db->restaurants;
+
+// Retrieves documents with a cuisine value of "Bakery", groups them by "borough", and
+// counts each borough's matching documents
+// start-match-group
+$pipeline = [
+    ['$match' => ['cuisine' => 'Bakery']],
+    ['$group' => ['_id' => '$borough', 'count' => ['$sum' => 1]]]
```

Comment:
```suggestion
    ['$group' => ['_id' => '$borough', 'count' => ['$sum' => 1]]],
```

Note: it's typically good practice to use trailing commas after all array elements, even though it's not required for the final element. I believe we actually enforce this in the PHPLIB coding standard, as it leads to more concise diffs when changing arrays.

This can apply to the `$pipeline` assignment below, too. I assume that is intentionally duplicated since it's displayed in a different location within the tutorial.